### DISCUSSION TITLE
TIFF Group 4 Compression

### DISCUSSION THREAD

**Discussion by frankracis-work** [SixLabors/ImageSharp#1988] [Ideas]
TIFF only supports decompression for Group 4, not compression.   I'd like it to compress as well.

I'm willing to do the work for this and submit a PR.

**Comment by JimBobSquarePants** [comment]
And we'd be more than happy to accept your help. That would be awesome! 😄

**Comment by brianpopow** [comment]
Just for the record: Thanks to @frankracis-work, since #1993 was merged to main, Tiff Group 4 Compression is now supported.
